types/src/wasm.rs

```diff
@@ -136,8 +140,12 @@ pub fn leader_index(seed: JsValue, participants: usize) -> usize {
         return 0;
     };
 
-    let round = Round::new(EPOCH, seed.view);
+    let round = Round::new(EPOCH, View::new(seed.view));
     let seed = Seed::new(round, signature);
 
-    select_leader::<Scheme, ()>(&vec![(); participants], round, Some(seed)).1 as usize
+    if round.view().get() == 1 {
+        round.epoch().get().wrapping_add(round.view().get()) as usize % participants
+    } else {
+        modulo(seed.encode().as_ref(), participants as u64) as usize
+    }
```

Comment:
It would be nice to export this, maybe? `Random` has a function to select a specific leader amongst a list of public keys, but no way to get the leader _index_ from a round/seed alone.